experimental/dds/tree2/src/test/snapshots/testTrees.ts

```diff
@@ -26,7 +27,10 @@ import {
 import { AllowedUpdateType, FieldKey, JsonableTree, UpPath, rootFieldKey } from "../../core";
 import { leaf, SchemaBuilder } from "../../domains";
 
-const factory = new SharedTreeFactory({ jsonValidator: typeboxValidator });
+const factory = new SharedTreeFactory({
+	jsonValidator: typeboxValidator,
+	forest: ForestType.Optimized,
```

Comment:
this change seems unrelated to this PR, and I'm not sure why its being made.

Reply:
removed unrelated change